src/main/java/uk/gov/digital/ho/hocs/document/service/MsgDocumentConverter.java

```
@@ -99,39 +107,109 @@ public MsgContents extractContents(InputStream inputStream) throws DocumentExcep
                 }
             }
-        } catch (RuntimeWorkerException | DocumentException | IOException e) {
+            return lines;
+        } catch (IOException e) {
             log.warn("Failed to Parse MSG Elements: {}", e.getMessage(), value(EVENT, DOCUMENT_CONVERSION_MSG_PARSE_FAILURE), value(EXCEPTION, e));
-            return false;
+            return List.of();
+        }
+    }
+
+    public void printContents(PDPageContentStream contentStream, List<String> lines, PDDocument pdf) {
+        try {
+            PDRectangle mediabox = pdf.getPage(0).getMediaBox();
+            float height = mediabox.getHeight() - 2 * margin;
+
+            int lineOnPage = 0;
+            contentStream.beginText();
+            contentStream.setFont(font, fontSize);
+            contentStream.setLeading(leading);
+            contentStream.newLineAtOffset(margin, 700);
+
+            for (String line: lines)
+            {
+                contentStream.showText(line);
+                contentStream.newLineAtOffset(0, -leading);
+                lineOnPage++;
+                if (lineOnPage * leading > height) {
+                    contentStream.endText();
+                    contentStream.close();
+
+                    PDPage page = new PDPage();
+                    pdf.addPage(page);
+                    contentStream = new PDPageContentStream(pdf, page, PDPageContentStream.AppendMode.APPEND, true, true);
+
+                    contentStream.beginText();
+                    contentStream.setFont(font, fontSize);
+                    contentStream.setLeading(leading);
+                    contentStream.newLineAtOffset(margin, 700);
+
+                    lineOnPage = 0;
+                }
+            }
+
+            contentStream.endText();
+            contentStream.close();
+        } catch (IOException e) {
+            log.warn("Failed to print MSG document: {}", e.getMessage(), value(EVENT, DOCUMENT_CONVERSION_MSG_PARSE_FAILURE), value(EXCEPTION, e));
```

Do we need to throw an exception here so the message ends up on the DLQ?